src/app/components/cookie-policy/cookie-policy-component.spec.ts

```diff
@@ -0,0 +1,56 @@
+import {Component} from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { CookiePolicyComponent } from './cookie-policy.component';
+
+describe('CookiePolicyComponentTest', () => {
+
+  @Component({selector: 'exui-app-header', template: ''})
+  class HeaderStubComponent {
+  }
+
+  @Component({selector: 'exui-app-footer', template: ''})
+  class FooterStubComponent {
+  }
+
+  @Component({
+    selector: `exui-app-host-dummy-component`,
+    template: `<exui-cookie-policy/>`
+  })
+  class TestDummyHostComponent {
+    public cookiePolicy: CookiePolicyComponent;
+  }
+  const testHostComponent = TestDummyHostComponent;
+  let component: CookiePolicyComponent;
+  let fixture: ComponentFixture<CookiePolicyComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ HeaderStubComponent, FooterStubComponent, CookiePolicyComponent ],
+      imports: [
+        RouterTestingModule
+      ]
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CookiePolicyComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+  it('should be created by angular', () => {
+    expect(fixture).not.toBeNull();
+  });
+  it('should include 4 security cookies', () => {
```

Comment:
Would be sensible, I think, to have a counterpart test that calls `component.cookiesByCat(component.SECURITY)` and then confirms that there are 4 items returned and that each has a `cat` equal to `component.SECURITY`.